### ISSUE TITLE
More basic examples

### ISSUE THREAD

**Issue by j256** [szabgab/code-maven.com#178]
Hey Gabor:

On your map page: https://code-maven.com/groovy-map you jump immediately into a map which has an array value.  Unless I'm missing a more simple page previously, I would think that starting with simpler name/value pairs would be more helpful.  You could then talk about name -> arrays in more complex maps page or just further down the page.

Thanks much.

**Comment by szabgab** [comment]
You are right. Unfortunately (?) I have not touched Groovy for a few years and I don't see that it will happen any time soon. So don't hold your breath for those simpler examples.
However, I will keep this issue open so if and when I get back to writing some Groovy I'll have a reminder for improvements.
